### PR TITLE
Release notes page

### LINKED ISSUE #636: [DOCUMENTATION] Revamp Release Notes Page
We would like the https://sparkdesignsystem.com/docs/release-notes page header to be changed to "Spark Releases". And all the other content on the page removed to be replaced with a list of links to the current and past Spark releases. The releases are here https://github.com/sparkdesignsystem/spark-design-system/releases

So, for example, the link for our current release would say "Fall 2018 - The Angular Release" and the link would be https://github.com/sparkdesignsystem/spark-design-system/releases/tag/v2.1.0.

Please open this Pull Request against our MASTER branch, Thank you so much! Happy Hacktoberfest!

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Added release name with its github release tag link as list.

Closes #636

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
@afebbraro thank you for the quick review

File: src/pages/docs/release-notes.hbs [hunk 1]
```diff
@@ -1,126 +1,28 @@
...
-      <li>
-        Added more useful explainer content for components.
-      </li>
-
+    <ul>
       <li>
-        Fixed an issue in the Navigation component where long
-        navigation items were breaking out of their container instead of wrapping.
+        <h2 class="drizzle-b-h2">
+          <a href="https://github.com/sparkdesignsystem/spark-design-system/releases/tag/v2.1.0">
```
Comment (RCopeland) [inline src/pages/docs/release-notes.hbs]: can you please add the class drizzle-b-Link to the links?

File: src/pages/docs/release-notes.hbs [hunk 2]
```diff
@@ -1,126 +1,28 @@
...
-
-      <li>
-        Fixed an error in the '.sprk-is-Disabled' class name for
-        Buttons where it was incorrectly named as '.is-Disabled'.
-      </li>
-
-      <li>
-        Added a section to the Spark Site for z-indexes called Layers.
+        <h2 class="drizzle-b-h2">
+          <a href="https://github.com/sparkdesignsystem/spark-design-system/releases/tag/v1.0.0">
```
Comment (RCopeland) [inline src/pages/docs/release-notes.hbs]: here too please

**Comment by arshadkazmi42** [comment]
@RCopeland Done with the changes 👍

**Pull request merged**